Assets/Scripts/SS3D/Systems/Health/BodyParts/HumanTorso.cs

```diff
@@ -58,6 +57,15 @@ protected override void AddInitialLayers()
 
         InvokeOnBodyPartLayerAdded();
     }
+    
+    protected override void InflictDamage(BodyLayer layer, DamageTypeQuantity damageTypeQuantity)
```

Comment:
I don't really see the custom logic in there. Why is this needed ?

Reply:
Please let reviewer resolve conversations. I get what you need to do, but I rather have this method stay private (at least not virtual). Can we instead have a virtual IsDetachable property on body parts ? true for most and false for torso ? And use this property with a if in the Inflict damage method (only detach if it's detachable)